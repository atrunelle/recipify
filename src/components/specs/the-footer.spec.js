import { mount } from '@vue/test-utils';
import TheFooter from '../the-footer';

describe('Component: the footer', () => {
  it('should match snapshot', () => {
    const wrapper = mount(TheFooter);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
