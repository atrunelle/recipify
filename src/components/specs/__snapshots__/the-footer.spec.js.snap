// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: the footer should match snapshot 1`] = `
<footer class="v-footer grey darken-3 theme--light" style="height: 32px;">
  <div class="layout row wrap aligh-center justify-center">
    <div class="flex xs12 text-xs-center white--text">
      <p class="my-2 body-1">
        Powered by Vue
      </p>
    </div>
  </div>
</footer>
`;
